src/components/structures/UserSettings.js

```diff
@@ -101,6 +108,7 @@ module.exports = React.createClass({
             middleOpacity: 1.0,
         });
         dis.unregister(this.dispatcherRef);
+        MatrixClientPeg.get().removeListener("RoomMember.membership", this._onInviteStateChange);
```

Comment:
```
$ grep -rn "MatrixClientPeg\.get()\." matrix-react-sdk/src/ | wc -l
     264
```

There are a lot of assumptions in other UI components that assume `MatrixClientPeg` is not `null`. These are generally safe assumptions because the UI component in question isn't shown when the user is not logged in: as is the case here. I would rather not increase the SNR any more than I need to. WDYT?